### ISSUE TITLE
problem

### ISSUE THREAD

**Issue by nestorlevi** [tomasz90/ESP32-OTA-Serial#1]
When I try to compile the example, I get the following error:
**undefined reference to `OTA::timer'**

**Comment by tomasz90** [comment]
There is no such function call in whole repo..

**Comment by waterbuffalo5** [comment]
I'm running into this as well. I tried using the example, and same thing. But I have had issues with Platformio and some OTA stuff in the past, so maybe related. 

```
Linking .pio\build\esp32dev\firmware.elf
c:/users/xxx/.platformio/packages/toolchain-xtensa-esp32/bin/../lib/gcc/xtensa-esp32-elf/8.4.0/../../../../xtensa-esp32-elf/bin/ld.exe: .pio/build/esp32dev/src/acd.cpp.o:(.literal._tartEPKcS1_jj[_ZN3OTA5startEPKcS1_jj]+0x3c): undefined reference to `OTA::timer'
collect2.exe: error: ld returned 1 exit status
*** [.pio\build\esp32dev\firmware.elf] Error 1
```


update
This is over my head, but chatgpt said to add this to my .cpp, and it seems to work. 
`TimerHandle_t OTA::timer = nullptr;`

**Comment by tomasz90** [comment]
You are right, I published the fix in version 1.0.9, thanks for letting me know!
